MIDIchlorians/AudioClipPlayer.swift

```diff
@@ -15,10 +15,11 @@ struct AudioClipPlayer {
 
     //creates ID and returns it to be mapped
     static func initAudioClip(audioDir: String, ext: String = "wav") -> UInt32? {
-
-        guard let soundURL = Bundle.main.url(forResource: audioDir, withExtension: ext) else {
+        guard let docsURL = FileManager.default.urls(for: .documentDirectory,
```

Comment:
Yup, seems to be working fine. I'm using the same url as the one you used in appDelegate